NonSucking.Framework.Serialization/Serializers/CtorSerializer.cs

```diff
@@ -13,9 +13,72 @@
 
 namespace NonSucking.Framework.Serialization.Serializers
 {
+
+    [Flags]
+    internal enum Initializer
+    {
+        None = 0,
+        Ctor = 1 << 1,
+        /// <summary>
+        /// Will enable <see cref="Ctor"/> aswell
+        /// </summary>
+        InitializerList = (1 << 2) | Ctor,
+        Properties = 1 << 3,
```

Comment:
```suggestion
        Ctor = 1 << 0,
        /// <summary>
        /// Will enable <see cref="Ctor"/> aswell
        /// </summary>
        InitializerList = (1 << 1) | Ctor,
        Properties = 1 << 2,
```
No reason to start at 2?

Reply:
Accidentically done it